### DISCUSSION TITLE
Menu => Content Menu Item show localization

### DISCUSSION THREAD

**Discussion by LMSSonos** [OrchardCMS/OrchardCore#8392] [Q&A]
Hi

My website has multiple languages and when I build the navigation and then link to a content menu item when searching and I have content items with the same name in multiple languages, there is no way to distinguish them. Is there an option to show as well the localization in the content menu item selector?

Thanks

**Comment by Skrypt** [comment]
Related with issue :  #5603
And PR closed with :  #5604

A solution remains to be implemented. The linked PR was not the proper solution but a second PR should take the discussion in consideration.
